hudi-common/src/test/java/org/apache/hudi/common/table/read/TestHoodieFileGroupReaderBase.java

```diff
@@ -136,7 +136,7 @@ public void testReadFileGroupInMergeOnReadTable(RecordMergeMode recordMergeMode,
           initialRecords, initialRecords);
 
       // Two commits; reading one file group containing a base file and a log file
-      List<HoodieRecord> updates = dataGen.generateUniqueUpdates("002", 50);
+      List<HoodieRecord> updates = dataGen.generateUniqueUpdates("002", 90);
```

Comment:
Any reason to change this?

Reply:
this is leftover from me debugging the issue with in memory index, will revert